core/src/main/resources/hudson/ProxyConfiguration/help-name.html

```diff
@@ -1,14 +1,22 @@
 <div>
-  If your Jenkins server sits behind a firewall and does not have the direct access to the internet,
-  and if your server JVM is not configured appropriately
-  (<a  href="https://docs.oracle.com/javase/8/docs/technotes/guides/net/properties.html">See JDK networking properties for more details</a>)
-  to enable internet connection, you can specify the HTTP proxy server name in this field to allow Jenkins
-  to install plugins on behalf of you. Note that Jenkins uses HTTPS to communicate with the update center to download plugins.
+  If your Jenkins server sits behind a firewall and does not have the direct
+  access to the internet, and if your server JVM is not configured appropriately
+  (
+  <a
+    href="https://docs.oracle.com/javase/8/docs/technotes/guides/net/properties.html"
+  >
+    See JDK networking properties for more details
+  </a>
+  ) to enable internet connection, you can specify the HTTP proxy server name in
+  this field to allow Jenkins to install plugins on behalf of you. Note that
+  Jenkins uses HTTPS to communicate with the update center to download plugins.
 
   <p>
-  Leaving this field empty
-  means Jenkins will try to connect to the internet directly.
+    Leaving this field empty means Jenkins will try to connect to the internet
+    directly.
+  </p>
 
   <p>
-  If you are unsure about the value, check the browser proxy configuration.
-</div>
\ No newline at end of file
+    If you are unsure about the value, check the browser proxy configuration.
+  </p>
+</div>
```

Comment:
I find such text spiting hard to translate in crowdin.

Reply:
this is on an outdated diff, not sure what you are referring to

Reply:
Strange. I still see this issue in my files changes view.
Maybe it's a github bug.

Reply:
The diff is outdated, but the text splitting is still present on the latest revision, diffed with master:

```diff
diff --git a/core/src/main/resources/hudson/ProxyConfiguration/help-name.html b/core/src/main/resources/hudson/ProxyConfiguration/help-name.html
index 04d3520563..eaba21d754 100644
--- a/core/src/main/resources/hudson/ProxyConfiguration/help-name.html
+++ b/core/src/main/resources/hudson/ProxyConfiguration/help-name.html
@@ -1,14 +1,22 @@
 <div>
-  If your Jenkins server sits behind a firewall and does not have the direct access to the internet,
-  and if your server JVM is not configured appropriately
-  (<a  href="https://docs.oracle.com/javase/8/docs/technotes/guides/net/properties.html">See JDK networking properties for more details</a>)
-  to enable internet connection, you can specify the HTTP proxy server name in this field to allow Jenkins
-  to install plugins on behalf of you. Note that Jenkins uses HTTPS to communicate with the update center to download plugins.
+  If your Jenkins server sits behind a firewall and does not have the direct
+  access to the internet, and if your server JVM is not configured appropriately
+  (
+  <a
+    href="https://docs.oracle.com/javase/8/docs/technotes/guides/net/properties.html"
+  >
+    See JDK networking properties for more details
+  </a>
+  ) to enable internet connection, you can specify the HTTP proxy server name in
+  this field to allow Jenkins to install plugins on behalf of you. Note that
+  Jenkins uses HTTPS to communicate with the update center to download plugins.
 
   <p>
-  Leaving this field empty
-  means Jenkins will try to connect to the internet directly.
+    Leaving this field empty means Jenkins will try to connect to the internet
+    directly.
+  </p>
 
   <p>
-  If you are unsure about the value, check the browser proxy configuration.
-</div>
\ No newline at end of file
+    If you are unsure about the value, check the browser proxy configuration.
+  </p>
+</div>
```

I agree, such a splitting of brackets is indeed difficult to work with, or just for reading.

Reply:
It's a bit odd but is it a blocker?

(for the actual file see https://github.com/timja/jenkins/blob/enable-prettier/core/src/main/resources/hudson/ProxyConfiguration/help-name.html)

Reply:
> It's a bit odd but is it a blocker?

No, I was just pointing out that the comment is relevant still.

Reply:
any reason you're holding back an approval?

Reply:
No particular one relating to this PR.